projects/baby_driver/src/main.c

```diff
@@ -0,0 +1,56 @@
+#include "can.h"
+#include "event_queue.h"
+#include "gpio.h"
+#include "interrupt.h"
+#include "log.h"
+
+#define CAN_DEVICE_ID 0x1
```

Comment:
We might want to define a new device ID for babydriver, we're not about to run out of them. Alternatively we could just use something large and silly à la 0xC0FFEE (but not exactly that since the python script is using it)

Reply:
Yeah 0x1 was just the default. Also does it matter? I thought these were for hardware and mainly for testing so the id shouldn't matter since the board wouldn't connect to other pieces. I might be wrong though.

Reply:
It probably won't connect to other boards, but I don't see a reason to inherently limit babydriver to not being able to connect to other boards.

Reply:
Sure. I'll leave it as 0x1 for now and we can define what ID it should have. Same with the python script